hooks/boilerplate.go.txt

```diff
@@ -0,0 +1,16 @@
+/*
+Copyright YEAR The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
```

Comment:
This doesn't really make sense in hooks without the actual commit hooks which reference it. Consider either copying the pre-commit hooks here, or inlining this in the build-ui.sh script.